airflow/secrets/environment_variables.py

```diff
@@ -30,8 +31,21 @@ class EnvironmentVariablesBackend(BaseSecretsBackend):
     """Retrieves Connection object and Variable from environment variable."""
 
     def get_conn_uri(self, conn_id: str) -> Optional[str]:
-        environment_uri = os.environ.get(CONN_ENV_PREFIX + conn_id.upper())
-        return environment_uri
+        """
+        Return URI representation of Connection conn_id
+        :param conn_id: the connection id
+        :return: deserialized Connection
+        """
+        warnings.warn(
```

Comment:
I tihnk we do not have to have the second warning here. It is already raised in "get_connection" when get_conn_uri is executed as fallback..

Reply:
if `get_connection` is doing the calling, it will find `get_conn_value` and it will not call this method.  so there should be no double warning.

but we can't just remove this method right?  we have to deprecate it.  so i think this warning needs to stay no?